src/app/optimizer/optimizer-data.service.ts

```diff
@@ -1,47 +1,49 @@
 import {Injectable} from '@angular/core';
 import {Http, Response} from '@angular/http';
-import {Subject, Observable} from 'rxjs/Rx';
+import {Subject, Observable, BehaviorSubject} from 'rxjs/Rx';
 
 @Injectable()
 export class OptimizerDataService {
   // TODO: Add these once backend can
   // initialInvestment: Number;
   // endValue: Number;
 
-  // Observable sources
-  private optimalAllocsSource = new Subject<Object>();
-  private sharpeRatioSource = new Subject<number>();
+  responseSubject: BehaviorSubject<Object> = new BehaviorSubject(
+    {"cumulative_returns":1.6650534305121512,
+     "optimal_allocations":{"FB":0.4510450475179859,"GOOG":0.5489549524820141,"AAPL":0},
+     "sharpe_ratio":0.5730332517669126});
 
-  // Observable streams
-  optimalAllocs$ = this.optimalAllocsSource.asObservable();
-  sharpeRatio$ = this.sharpeRatioSource.asObservable();
+  subject: BehaviorSubject<Object>;
```

Comment:
Consider renaming "subject" to something more informative, like "modelSubject" or "formDataSubject"?